### DISCUSSION TITLE
Plz can you add Scrollbar design in customCTk

### DISCUSSION THREAD

**Discussion by KamdemDarren** [TomSchimansky/CustomTkinter#159] [Ideas]
plz can you add the designed Scrollbar version in customCTk
and Thks for CTkComboxBox it is  working fine :) :)
Good work plz keepup

**Comment by Haadi-A** [comment]
Yh, I tried asking for that. as you can see in my question. Tkinter's scrollbar is just so much work to deal with, so it would be nice if we had a customtkinter scroll bar which can even scroll actual root without having to bind it to a canvas and do all those wired stuff just to get your tkiner window scrollable.
